.gitignore

```diff
@@ -4,4 +4,5 @@ __output/
 .DS_Store
 .idea
 npm-debug.log
-/tests/browser/**/www
\ No newline at end of file
+/tests/browser/**/www
+package-lock.json
```

Comment:
it's not reccomended to gitignore `package-lock.json`, see https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5 for example